mail_factory/mails.py

```diff
@@ -79,6 +79,30 @@ def get_template_part(self, part, lang=None):
         # return the list of templates path candidates
         return templates
 
+    def get_text_content(self, html_content):
+        """Return text content generated by html2text.
+
+        HTML2Text can be configured by having the following in your settings:
+
+        HTML2TEXT = {
+            'unicode_snob': True,
+            'ignore_images': True,
+            'single_line_break': True,
+            ...
+        }
+        More options can be found at:
+        https://github.com/Alir3z4/html2text/blob/master/docs/usage.md
+        """
+        html2text_settings = getattr(settings, "HTML2TEXT", None)
```

Comment:
Either use the APP_NAME as prefix, either make an APP_NAME dict setting which is the new way to go.